win32/Lib/pywin32_bootstrap.py

```diff
@@ -17,5 +17,18 @@
     # https://docs.python.org/3/reference/import.html#path-attributes-on-modules
     for path in pywin32_system32.__path__:
         if os.path.isdir(path):
-            os.add_dll_directory(path)
+            try:
+                # First try the preferred method
+                os.add_dll_directory(path)
+            except Exception:
+                # If anything fails, try to modify PATH if it exists
+                try:
+                    if "PATH" in os.environ:
+                        os.environ["PATH"] = path + os.pathsep + os.environ["PATH"]
+                    else:
+                        # If PATH doesn't exist, just create it
+                        os.environ["PATH"] = path
+                except Exception:
```

Comment:
What kind of exception do you expect here? It can't be a `KeyError` since you check for that.
`AttributeError` on `os.environ` ??

Reply:
See https://github.com/mhammond/pywin32/pull/2596#discussion_r2072225639 instead